spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'order'
+describe Order do
+  it 'can make an order' do
+    order = Order.new
+    expect(order).to be_a_kind_of(Order)
+  end
+
+  it 'can select number of several dishes' do
+    order = Order.new
+    menu = Menu.new
+  expect(order.select("Sushi",3)).to eq([{:dish=>"Sushi", :price=>7.0, :quantity=>3}])
+  end
+
+  it 'gives the total matches with the sum of various dishes in my order' do
+    order = Order.new
+    menu = Menu.new
```

Comment:
I'm not sure this menu is needed here? Ideally this menu could be mocked so that we are keeping our objects loosely coupled.